DeepCat.js

```diff
@@ -8,7 +8,8 @@
 ( function( $, mw ) {
 	var DeepCat = {},
 		keyString = 'deepcat:',
-		maxDepth = 15,
+		defaultDepth = 15,
+		maxDepth = 20,
```

Comment:
Ok, thanks. There should be a default here in the frontend code, but a limit doesn't make much sense in my opinion, especially if the backend is fine with high numbers. If the user asks for `~100`, why should this silently be truncated to 20? That's why I suggest to remove this.